bucket/xdman.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": "7.2.11",
-    "description": "A powerful FOSS downloader supports protocols as HTTP,HTTPS,FTP,MPEG-DASH,HLS,HDS and features as seamlessly integration with browsers, monitoring and capture, broken/dead tasks resumption.",
+    "description": "Xtreme Download Manager (XDM) is a powerful tool to increase download speeds up to 500%, save videos from popular video streaming websites, resume broken/dead downloads, schedule and convert downloads.",
```

Comment:
```suggestion
    "description": "A powerful tool to increase download speeds up to 500%, save videos from popular video streaming websites, resume broken/dead downloads, schedule and convert downloads.",
```

Reply:
I thought having "Xtreme Download Manager" in the description would make the meaning of XDM clearer

Reply:
Okay, keep 'Xtreme Download Manager' but remove '(XMD)'

Reply:
Done